src/Tools/BuildBoss/Program.cs

```diff
@@ -30,12 +26,14 @@ private static bool MainCore(string[] args)
         {
             string repositoryDirectory = null;
             string configuration = "Debug";
+            string primarySolution = null;
             List<string> solutionFiles;
 
             var options = new OptionSet
             {
                 { "r|root=", "The repository root", value => repositoryDirectory = value },
-                { "c|configuration=", "Build configuration", value => configuration = value }
+                { "c|configuration=", "Build configuration", value => configuration = value },
+                { "p|primary=", "Primary solution file name", value => primarySolution = value },
```

Comment:
>p [](start = 19, length = 1)

`s|solution`?

Reply:
Perhaps describe it as "solution that builds all projects". It's not clear what primary means.

---
In reply to: [276783979](https://github.com/dotnet/roslyn/pull/35066#discussion_r276783979) [](ancestors = 276783979)

Reply:
I updated the description 👍